src/main/java/uk/ac/ebi/biostudies/api/util/LinkInfo.java

```diff
@@ -0,0 +1,62 @@
+package uk.ac.ebi.biostudies.api.util;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.node.ArrayNode;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class LinkInfo {
```

Comment:
`Info` is redundant and we want to distinguish this class from normal links. Lets call this `ExtractedLink`